src/main/java/step3/racingcarWinner/RacingGame.java

```diff
@@ -0,0 +1,97 @@
+package step3.racingcarWinner;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RacingGame {
+
+  private static String inputName;
+  private static int inputTryCount;
+  private static int DEFAULT_CAR_LOCATION = 0;
```

Comment:
상수는 `static final` 필드입니다. 아래의 코드로 개선해 보면 어떨까요?

```suggestion
  private static final int DEFAULT_CAR_LOCATION = 0;
```

Reply:
final로 선언하여 override 되는것을 막음
수정하겠습니다!